### PR TITLE
[ADD] stock_picking_fillwithstock

### PR DESCRIPTION
**Pull request by i-vyshnevska**
Addons add a button which allows quickly fill stock picking with everything in a bin

### CODE REVIEW

File: stock_picking_fillwithstock/models/stock.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                    }
+                else:
+                    products[quant.product_id.id]['product_uom_qty'] += qty
+        move_obj = self.env['stock.move']
+        if not available:
+            raise Warning(_('Nothing to move'))
+        for data in products.values():
+            move_obj.create(data)
+        self.action_confirm()
+        self.action_assign()
```
Comment (sebalix) [inline stock_picking_fillwithstock/models/stock.py:56]: You should return something (at least `True` to be compatible with XML-RPC).

File: stock_picking_fillwithstock/views/stock.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form.fillwithstock</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
+            <button name="action_confirm" position="before">
+                <button name="button_fillwithstock" states="draft" string="Fill with stock" type="object" class="oe_highlight" groups="base.group_user" />
```
Comment (sebalix) [inline stock_picking_fillwithstock/views/stock.xml]: `base.group_user` or `stock.group_stock_user`? There is no guarantee that the first could access to the quants, moves, etc, better to switch to the second I think.

**Comment by rousseldenis** [comment]
@i-vyshnevska Have you ever looked at this module ? https://github.com/OCA/stock-logistics-warehouse/tree/12.0/stock_move_location

I think this is for the same purpose.

**Comment by i-vyshnevska** [comment]
> @i-vyshnevska Have you ever looked at this module ? https://github.com/OCA/stock-logistics-warehouse/tree/12.0/stock_move_location
> 
> I think this is for the same purpose.

thanks, @rousseldenis, I haven't checked that one, on first look introduced addons works on a more granular level

**Comment by rousseldenis** [comment]
> > @i-vyshnevska Have you ever looked at this module ? https://github.com/OCA/stock-logistics-warehouse/tree/12.0/stock_move_location
> > I think this is for the same purpose.
> 
> thanks, @rousseldenis, I haven't checked that one, on first look introduced addons works on a more granular level

Maybe you could extend it or complete it in order to not duplicating code or helping people not to be doubtful on which one to choose

**Comment by i-vyshnevska** [comment]
feature moved to stock_move_location
https://github.com/OCA/stock-logistics-warehouse/pull/885

**Pull request closed without merging**